book/3 - Configuring settings.ini/4 - Server.html

```diff
@@ -0,0 +1,73 @@
+<p>
+	Basic server settings are configured here.
+</p>
+
+<h5>Available Options</h5>
+
+<figure>
+<table>
+  <tr>
+    <th>Variable</th>
+    <th>Meaning</th>
+    <th>Default</th>
+  </tr>
+  <tr>
+    <td>Description</td>
+    <td>The description which appears in the Minecraft client server list</td>
+    <td>Cuberite - in C++!</td>
+  </tr>
+  <tr>
+    <td>AllowBungeeCord</td>
+    <td>See {{5.5 - Bungeecord}}</td>
+    <td>0</td>
+  </tr>
+  <tr>
+    <td>MaxPlayers</td>
+    <td>The maximum amount of players allowed to join the server</td>
+    <td>100</td>
+  </tr>
+  <tr>
+    <td>HardcoreEnabled</td>
+    <td>In Hardcore mode, players who die are banned from the server. 0 is standard, 1 is Hardcore</td>
+    <td>0</td>
+  </tr>
+  <tr>
+    <td>AllowMultiLogin</td>
+    <td>TODO</td>
+    <td>0</td>
+  </tr>
+  <tr>
+    <td>Ports</td>
+    <td>Which ports should Cuberite listen to</td>
+    <td>25565</td>
+  </tr>
+  <tr>
+    <td>DefaultViewDistance</td>
+    <td>The maximum view distance a player may have, in chunks. Lower this to improve performance</td>
+    <td>10</td>
+  </tr>
+  <tr>
+    <td>MaxChunkRAM</td>
+    <td>
+	The (soft) maximum amount of RAM which can be used by chunk data. 
```

Comment:
I don't understand what you mean here - what could go wrong with long paragraphs?